requirements.json

```diff
@@ -177,5 +177,30 @@
         "sha256sum": "d65a1d1a050e4fd7912c18468954f64db824a1e1b621235a153d010beae14757"
       }
     }
+  },
+  "kompose": {
+    "name": "Kompose",
+    "description": "Kompose takes a Docker Compose file and translates it into Kubernetes or OpenShift resources",
```

Comment:
@cdrage Could we rephrase "Kompose takes a Docker Compose file and translates it into Kubernets or OpenShift resources" form what it does to what it is. I am asking because all other components do that. It would be something like: "A tool to translate Docker Compose file into  Kubernets or OpenShift resources".
Does that sound good/bad?